otlp/traces.go

```diff
@@ -61,12 +61,18 @@ func TranslateTraceRequest(request *collectorTrace.ExportTraceServiceRequest, ri
 					"type":             spanKind,
 					"span.kind":        spanKind,
 					"name":             span.Name,
-					"duration_ms":      float64(span.EndTimeUnixNano-span.StartTimeUnixNano) / float64(time.Millisecond),
 					"status_code":      statusCode,
 					"span.num_links":   len(span.Links),
 					"span.num_events":  len(span.Events),
 					"meta.signal_type": "trace",
 				}
+				duration := float64(span.EndTimeUnixNano-span.StartTimeUnixNano) / float64(time.Millisecond)
+				if duration < 0 {
+					duration = 0
+					eventAttrs["meta.negative_duration"] = true
```

Comment:
Verging on bikeshedding, but why not a negative value like -1? It's just a little strange (to me at least) to write an event as having a negative duration when the duration isn't negative.

Reply:
Mainly because when you do that, Honeycomb interprets it as a Very Large positive number.

I did consider doing this:

`eventAttrs["meta.negative_duration"] = -duration`

But thought that had the potential to be even more confusing.

Another option would be a different name. 'meta.end_before_start=true`?

Reply:
Ohhhh, I see. Well, that's fun. Carry on, LGTM!

Reply:
Carrying on with bikeshedding I suppose, but what about something like `meta.invalid_duration`?